src/PBot.Tests/UpdateRepoLabelsTests.cs

```diff
@@ -31,153 +30,67 @@ public class UpdateRepoLabelsTests
                 { "Resolution: Can't Reproduce", "Withdrawn: Invalid" },
                 { "Resolution: Duplicate", "Withdrawn: Duplicate" },
                 { "Resolution: Won't Fix", "Withdrawn: Won't Fix" },
+                { "State: Approved", null },
                 { "State: Prioritized", null },
+                { "State: Review", null },
                 { "Urgent", null },
+                { "Type: Breaking Change", "Tag: Breaking Change" },
                 { "wontfix", "Withdrawn: Won't Fix" },
             };
 
         [Test, Explicit("Performs the actual sync for now")]
-        public void SyncAllRepos()
+        [TestCase("Particular", new[] { "Operations.Licensing", "PlatformDevelopment", "ProductionTests" })]
+        public async void SyncLabels(string org, string[] privateRepos)
         {
-            var client = GitHubClientBuilder.Build();
-            var sourceRepo = "RepoStandards";
-
+            Console.Out.WriteLine($"Syncing labels for {org}...");
 
-            var labelsToSync = client.Issue.Labels.GetForRepository(Organization, sourceRepo).Result;
-
-            //go through all repos
-            var repos = new[]
+            var client = GitHubClientBuilder.Build();
+            var templateLabels = await client.Issue.Labels.GetForRepository(org, "RepoStandards");
+            var syncs = (await client.Repository.GetAllForOrg(org))
+                .Where(repo => !repo.Private)
```

Comment:
This won't include `PlatformDevelopment`. Should we whitelist private repos too?